code/modules/antagonists/nuclear_operative/nuclear_operative.dm

```diff
@@ -51,6 +51,9 @@
 		else
 			H.equip_if_possible(new /obj/item/tank/emergency_oxygen/extended(H), SLOT_L_STORE)
 
+		if(H.traitHolder?.hasTrait("mildly_mutated"))
+			H.traitHolder.removeTrait("mildly_mutated")
```

Comment:
I suspect this won't solve the issue, as the trait being added will activate (and reinforce) the gene, but removing it doesn't do the opposite.